web/src/engine/osk/src/input/gestures/specsForLayout.ts

```diff
@@ -655,13 +711,97 @@ export const ModipressEndModel: GestureModel<any> = {
     {
       model: {
         ...ModipressContactEndModel,
-        itemChangeAction: 'reject'
+        itemChangeAction: 'reject',
+        pathInheritance: 'full'
       }
     }
   ],
   resolutionAction: {
-    type: 'complete',
+    type: 'chain',
+    // Because SHIFT -> CAPS multitap is a thing.  Shift gets handled as a modipress first.
+    // Modipresses resolve before multitaps... unless there's a model designed to handle & disambiguate both.
+    next: 'modipress-multitap-start',
+    // Key was already emitted from the 'modipress-start' stage.
     item: 'none'
   }
 }
+
+export function modipressMultitapStartModel(params: GestureParams): GestureModel<KeyElement> {
+  return {
+    id: 'modipress-multitap-start',
+    resolutionPriority: 6,
+    contacts: [
+      {
+        model: {
+          ...ModipressContactStartModel,
+          pathInheritance: 'reject',
+          allowsInitialState(incomingSample, comparisonSample, baseItem) {
+            if(incomingSample.item != baseItem) {
+              return false;
+            }
+            // TODO:  needs better abstraction, probably.
+
+            // But, to get started... we can just use a simple hardcoded approach.
+            const modifierKeyIds = ['K_SHIFT', 'K_ALT', 'K_CTRL'];
```

Comment:
How about `K_NUMERIC`, `K_SYMBOL`?

Reply:
Done in #9876, which aims to more comprehensively cover modipresses.